tests/unit/store/utility.spec.ts

```diff
@@ -0,0 +1,282 @@
+      }
+
+      // 転写されている
+      ["お", "しょ"].forEach((moraText) => {
+        expect(findMora(result, moraText)).toEqual(findMora(before, moraText));
+      });
+
+      // 転写されていない
+      ["あ", "しゃ", "き", "きゅ"].forEach((moraText) => {
+        expect(findMora(result, moraText)).not.toEqual(
+          findMora(before, moraText)
+        );
+      });
+    });
+  });
+
+  describe("isAccentPhrasesTextDifferent", () => {
+    it("アクセントフレーズのテキストが異なる場合、trueを返す", () => {
+      const accentPhrases1 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const accentPhrases2 = [
+        createDummyAccentPhrase(["か", "き", "く"]),
+        createDummyAccentPhrase(["け", "こ"]),
+      ];
+      const result = isAccentPhrasesTextDifferent(
+        accentPhrases1,
+        accentPhrases2
+      );
+      expect(result).toBe(true);
+    });
+
+    it("アクセントフレーズのテキストが同じ場合、falseを返す", () => {
+      const accentPhrases1 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const accentPhrases2 = [
+        createDummyAccentPhrase(["あ", "い", "う"]),
+        createDummyAccentPhrase(["え", "お"]),
+      ];
+      const result = isAccentPhrasesTextDifferent(
+        accentPhrases1,
+        accentPhrases2
+      );
+      expect(result).toBe(false);
+    });
+  });
+
+  test("buildAudioFileNameFromRawData", () => {
+    const fileNamePattern = "テストパターン";
+    const vars = {
+      index: 1,
+      characterName: "四国めたん",
+      text: "テストテキスト",
+      styleName: "ツンツン",
+      date: currentDateString(),
+    };
+    const result = buildAudioFileNameFromRawData(fileNamePattern, vars);
+    expect(result).toBe("テストパターン");
+  });
+
+  test("getToolbarButtonName", () => {
+    expect(getToolbarButtonName("PLAY_CONTINUOUSLY")).toBe("連続再生");
+    expect(getToolbarButtonName("STOP")).toBe("停止");
+    expect(getToolbarButtonName("EXPORT_AUDIO_SELECTED")).toBe(
+      "選択音声を書き出し"
+    );
+    expect(getToolbarButtonName("EXPORT_AUDIO_ALL")).toBe("全部書き出し");
+    expect(getToolbarButtonName("EXPORT_AUDIO_CONNECT_ALL")).toBe(
+      "音声を繋げて書き出し"
+    );
+    expect(getToolbarButtonName("存在しないタグ" as ToolbarButtonTagType)).toBe(
+      undefined
+    );
+  });
+
+  describe("createKanaRegex", () => {
+    it("includeSeparationがtrueの場合、読点とクエスチョンも含む", () => {
+      const regex = createKanaRegex(true);
+      expect(regex.test("あいうえお、")).toBe(true);
+      expect(regex.test("かきくけこ？")).toBe(true);
+    });
+
+    it("includeSeparationがfalseの場合、読点とクエスチョンを含まない", () => {
+      const regex = createKanaRegex(false);
+      expect(regex.test("あいうえお、")).toBe(false);
+      expect(regex.test("かきくけこ？")).toBe(false);
+    });
+  });
+
+  test("convertHiraToKana", () => {
+    expect(convertHiraToKana("あいうえお")).toBe("アイウエオ");
+    expect(convertHiraToKana("かきくけこ")).toBe("カキクケコ");
+    expect(convertHiraToKana("さしすせそ")).toBe("サシスセソ");
```

Comment:
ちょっと色んなサンプル追加させていただきます！
```suggestion
    expect(convertHiraToKana("さしすせそ")).toBe("サシスセソ");
    expect(convertHiraToKana("がぱをんー")).toBe("ガパヲンー");
```